charts/nri-kube-events/values.yaml

```diff
@@ -24,6 +24,8 @@
 
 agentHTTPTimeout: "30s"
 
+proxy: {}
```

Comment:
Can you add some comments commented out?
Only emplaining by which container it is used and why it can be useful

Reply:
sure! let me fix this